components/Cards.js

```diff
@@ -20,3 +23,83 @@
 // Add a listener for click events so that when a user clicks on a card, the headline of the article is logged to the console.
 //
 // Use your function to create a card for each of the articles, and append each card to the DOM.
+
+axios.get('https://lambda-times-api.herokuapp.com/articles')
+     
+
+        .then(({data}) => {
+            console.log(data)
+          
+            
+            data.articles.bootstrap.forEach(item =>{
+                 let ncard=cardCreator(item);
+                document.querySelector('.cards-container').append(ncard);   
+            })
+            data.articles.javascript.forEach(item =>{
+                let ncard=cardCreator(item);
+                document.querySelector('.cards-container').append(ncard);   
+            })
+            data.articles.jquery.forEach(item =>{
+                let ncard=cardCreator(item);
+                document.querySelector('.cards-container').append(ncard);   
+            })
+            data.articles.node.forEach(item =>{
+                let ncard=cardCreator(item);
+                document.querySelector('.cards-container').append(ncard);   
+            })
+            data.articles.technology.forEach(item =>{
+                let ncard=cardCreator(item);
+                document.querySelector('.cards-container').append(ncard);   
+            })
+        
+           
+        })
+        .catch(err => {
+            console.log(err)
+        })
+        
+        
+
+        function cardCreator(info){
```

Comment:
Please don't indent this function definition. It's harder to tell that the function is not being defined inside another function when it is indented like this. It should be aligned with the `axios.get` line